src/App.jsx

```diff
@@ -1,50 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import './App.scss';
 
-// Let's talk about using index.js and some other name in the component folder.
-// There's pros and cons for each way of doing this...
-// OFFICIALLY, we have chosen to use the Airbnb style guide naming convention. 
 // Why is this source of truth beneficial when spread across a global organization?
 import Header from './Components/Header';
 import Footer from './Components/Footer';
 import Form from './Components/Form';
 import Results from './Components/Results';
 
-class App extends React.Component {
+const App = () => {
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      data: null,
-      requestParams: {},
-    };
-  }
+  const [data, setData] = useState(null);
+  const [reqParams, setReqParams] = useState ({});
+  const [loading, setLoading] = useState(false);
 
-  callApi = (requestParams) => {
-    // mock output
-    const data = {
-      count: 2,
-      results: [
+  const callApi = (reqParams) => {
+    setLoading(true);
```

Comment:
You set loading true, make sure you reset it to false when you have finished loading.